mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -2291,4 +2240,66 @@
     <string name="a11y_heading">%s, Heading</string>
+
+    <!-- Translations feature-->
+
+    <!-- Automatic translation preference screen -->
+    <!-- Title of the automatic translation preference screen that will appear on the toolbar.-->
+    <string name="automatic_translation_toolbar_title_preference">Automatic translation</string>
+    <!-- Screen header presenting the automatic translation preference feature. It will appear under the toolbar. -->
+    <string name="automatic_translation_header_preference">Select a language to manage ”always translate“ and ”never translate“ preferences.</string>
+
+    <!-- Automatic translation options preference screen -->
+    <!-- Preference option for offering to translate. Radio button title text.-->
+    <string name="automatic_translation_option_offer_to_translate_title_preference">Offer to translate (default)</string>
+    <!-- Preference option for offering to translate. Radio button summary text. The first parameter is the name of the app defined in app_name (for example: Fenix)-->
+    <string name="automatic_translation_option_offer_to_translate_summary_preference">%1$s will offer to translate sites in this language.</string>
+    <!-- Preference option for always translate. Radio button title text. -->
+    <string name="automatic_translation_option_always_translate_title_preference">Always translate</string>
+    <!-- Preference option for always translate. Radio button summary text. The first parameter is the name of the app defined in app_name (for example: Fenix)-->
+    <string name="automatic_translation_option_always_translate_summary_preference">%1$s will translate this language automatically when the page loads.</string>
+    <!-- Preference option for never translate. Radio button title text.-->
+    <string name="automatic_translation_option_never_translate_title_preference">Never translate</string>
+    <!-- Preference option for never translate. Radio button summary text. The first parameter is the name of the app defined in app_name (for example: Fenix)-->
+    <string name="automatic_translation_option_never_translate_summary_preference">%1$s will never offer to translate sites in this language.</string>
+
+    <!-- Never translate site preference screen -->
+    <!-- Title of the never translate site preference screen that will appear on the toolbar.-->
+    <string name="never_translate_site_toolbar_title_preference">Never translate these sites</string>
+    <!-- Screen header presenting the never translate site preference feature. It will appear under the toolbar. -->
+    <string name="never_translate_site_header_preference">To add a new site: Visit it and select “Never translate this site” from the translation menu.</string>
```

Comment:
@ohall-m 
The option is plural (string right above): "Never translate **these sites**". Which one is correct?

Reply:
Actually, the string above is a "title". I don't see a command `Never translate this site` though.

Reply:
https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?type=design&node-id=7202-99696&mode=design&t=USuT07aPzw2YWuoL-0
It is used here 
<string name="translation_option_bottom_sheet_never_translate_site">Never translate this site</string>

Reply:
But there is [no string exposed with that text](https://transvision.flod.org/?recherche=never+translate+this+site&repo=android_l10n&sourcelocale=en-US&locale=it&search_type=strings_entities). Is that code still not done?

Reply:
It will be added with this pr https://github.com/mozilla-mobile/firefox-android/pull/4810/files#diff-b24d2928f9f4594421c030b62d3e5e070c9fd5600c871ff9a8c24a518fd02f96 
The string was, until now, in the static_strings file.

Reply:
Hi @flodolo, is the string we are looking for `never_translate_site_header_preference`? It looks like that was added in [Bug 1856970](https://github.com/mozilla-mobile/firefox-android/commit/10b8ba0d36b1d07bf1dd930ab29398186ae9fa91) with usage.

Reply:
No, it's OK as is. I was confused and wondered if there was an inconsistency in the UI, but it's just a string not exposed for localization yet.